routines/SAMIFORM.m

```diff
@@ -1,4 +1,4 @@
-SAMIFORM ;ven/gpl - ielcap: form library ; 4/23/19 10:08am
+SAMIFORM ;ven/gpl - ielcap: form library ; 2019-08-01T15:42Z
```

Comment:
Is this change to the datetime stamp what we should be using going forward?

Reply:
I'm not sure what controls those timestamps. I think we are inconsistent. Something we should look at going forward. (after patch 1)